### DISCUSSION TITLE
Numeric和symbolic返回中文主鍵盤怎麼設置？

### DISCUSSION THREAD

**Discussion by Gs-Linguashop** [imfuxiao/Hamster#514] [Q&A]
好像Numeric和Symbolic鍵盤上都只能返回英文鍵盤。由於不適應九宮格的數字和符號鍵盤或者分類鍵盤，我將中文鍵盤上直接連結了默認的英文數字/符號鍵盤，但是這樣就沒法直接返回中文主鍵盤了。請問有沒有默認的中文數字/符號鍵盤，不要九宮格和分類的，就要正常的。（可能已經有了，就是我在Wiki上沒找到，有沒有好心人可以告訴我對應的KeyboardType叫什麼？）

**Comment by imfuxiao** [comment]
见[「手册-如何使用类系统原生的数字键盘及符号键盘」](https://ihsiao.com/apps/hamster/docs/Questions.html#%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8%E7%B1%BB%E7%B3%BB%E7%BB%9F%E5%8E%9F%E7%94%9F%E7%9A%84%E6%95%B0%E5%AD%97%E9%94%AE%E7%9B%98%E5%8F%8A%E7%AC%A6%E5%8F%B7%E9%94%AE%E7%9B%98)

**Comment by Gs-Linguashop** [comment]
您好，

感谢您百忙之中的回复。

我这边使用提供的中文26键它的数字及符号键盘是服从您给的链接的描述的行为的。但是使用任何自定义键盘时（App安装自带或者我自己编辑的），键盘设置里的数字九宫格和分类符号键盘的关闭并不起作用。

不知道有没有办法在自定义键盘里也能调用
类似系统的数字键盘和符号键盘呢？或者让键盘设置里的数字九宫格和分类符号键盘开关能够在自定义键盘里也起作用呢？

也有可能是设置里的数字九宫格和分类符号键盘开关是能够在自定义键盘里起作用的，只是我这边运行的bug。您看您那边是怎样运行的呢？

非常感谢您的工作和答疑，
Ziqi Zhou
